stuf_zds_payments/plugin.py

```diff
@@ -0,0 +1,132 @@
+import logging
+from typing import Any
+
+from django.utils.translation import gettext_lazy as _
+
+from openforms.registrations.contrib.stuf_zds.plugin import (
+    StufZDSRegistration,
+    ZaakOptionsSerializer,
+)
+from openforms.registrations.contrib.stuf_zds.utils import flatten_data
+from openforms.registrations.registry import register
+from openforms.submissions.models import Submission
+from openforms.variables.service import get_static_variables
+from rest_framework import serializers
+from stuf.stuf_zds.client import ZaakOptions
+
+from .client import get_client
+from .registration_variables import register as variables_registry
+
+logger = logging.getLogger(__name__)
+
+PLUGIN_IDENTIFIER = "stuf-zds-create-zaak:ext-utrecht"
+
+
+def default_payment_status_update_mapping() -> list[dict[str, str]]:
```

Comment:
(for when we port this to OF core)

I think that `TypedDict` is more appropriate here - now key/value can be any string, but if you do:

```py
class PaymentVariableMapping(TypedDict):
    form_variable: str
    stuf_name: str
```
you get much more accurate types